account.html

```diff
@@ -150,12 +150,12 @@ <h3>
         </div>
     </div>
 
-    <div class="block-password">
-        <div class="block-password__body">
+    <div class="user-form">
+        <div class="user-form__body">
             <div class="arrow-back">
                 <img src="img/pageAccount/arrow.svg" alt="arrow">
             </div>
-            <form class="block-password__form valid">
+            <form class="user-form__form valid">
                 <input class="valid-password" placeholder="Enter old password" type="text">
```

Comment:
```suggestion
                <input class="valid-password" placeholder="Enter old password" type="password">
```
in other forms do the same